pkg/proxy/userspace/proxier.go

```diff
@@ -972,6 +1015,13 @@ func (proxier *Proxier) iptablesHostNodePortArgs(nodePort int, protocol api.Prot
 	return args
 }
 
+// Build a slice of iptables args for an from-non-local public-port rule.
+func (proxier *Proxier) iptablesNonLocalNodePortArgs(nodePort int, protocol api.Protocol, proxyIP net.IP, proxyPort int, service proxy.ServicePortName) []string {
+	args := iptablesCommonPortalArgs(nil, false, false, proxyPort, protocol, service)
+	args = append(args, "-m", "state", "--state", "NEW", "-j", "ACCEPT")
```

Comment:
please add a `"-m", "comment", "--comment", service.String()` - it makes for easier grep